Description: link mapnik against dl
Author: Jérémy Lal <kapouer@melix.org>
Forwarded: https://github.com/mapnik/mapnik/pull/1998
Last-Update: 2013-08-30
--- a/src/build.py
+++ b/src/build.py
@@ -106,6 +106,7 @@
             mapnik_lib_link_flag += ' -Wl,-h,%s' %  mapnik_libname
     else: # Linux and others
         mapnik_lib_link_flag += ' -Wl,-rpath-link,. -Wl,-soname,%s' % mapnik_libname
+	lib_env['LIBS'].append('dl')
 
 source = Split(
     """
